xbmc/utils/EGLUtils.h

```diff
@@ -14,6 +14,9 @@
 #include <string>
 #include <vector>
 
+// needed before including egl.h as it's defined in gbm.h implicitly
+// this allows us to support generic types using EGL_NO_X11
+#undef __GBM__
```

Comment:
Can't really rely on this in a header, since `egl.h` might have already been included.

Reply:
Yea, I'm not really sure how to solve that :/

Reply:
You'll want to remove this undef in your latest version, right?

Reply:
removed thanks :)